### PR TITLE
As User - I want to use a new "Help" menu in the top bar

### PR DESCRIPTION
**Pull request by gitstart**
Closes https://github.com/amplication/amplication/issues/5007
### DEMO_VIDEO
Screenshot 2023-02-10 at 15 30 51

https://www.loom.com/share/54d89adaf7cf4810a4187cc45f33a389

### CODE REVIEW

File: packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx [hunk 1]
```diff
@@ -27,6 +27,20 @@ const CLASS_NAME = "workspace-header";
...
+type HelpMenuItem = {
+  name: string;
+  url: string | null;
+};
+
+const HELP_MENU_LIST: HelpMenuItem[] = [
+  { name: "Docs", url: "https://docs.amplication.com" },
+  {
+    name: "Technical Support",
+    url: "https://discord.com/channels/757179260417867879/1063337571184279582",
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx:39]:
```suggestion
    url: "https://amplication.com/discord",
```

File: packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.scss [hunk 2]
```diff
@@ -128,6 +128,65 @@ $select-menu-max-width: 300px;
...
+    .select-menu__summary.amp-button.amp-button--text {
+      color: var(--black80);
+      &.highlight,
+      &:hover,
+      &:focus,
+      &:active {
+        color: var(--black100);
+      }
+    }
+
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.scss:144]: @EugeneTseitlin can you please have a look at this?
Do we need all the CSS locally? I expect the styles to be inherited from the component directly from the design system

File: packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx [hunk 3]
```diff
@@ -278,6 +277,41 @@ const WorkspaceHeader: React.FC<{}> = () => {
...
+            className={`${CLASS_NAME}__help_popover__menu`}
+          >
+            <SelectMenuModal align="right">
+              <SelectMenuList>
+                {HELP_MENU_LIST.map((route: HelpMenuItem, index) => (
+                  <SelectMenuItem
+                    closeAfterSelectionChange
+                    onSelectionChange={() => {
+                      !route.url && handleContactUsClick();
+                    }}
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx:295]: This solution is OK only for the current situation where we have a single item with a callback, but we need to implement a more generic solution.

we can use `itemData` on SelectMenuItem to pass a name of a command and then in a callback function trigger a function based on the command name

{ name: "Contact Us", url: null, itemData: COMMAND_CONTACT_US },

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
Thanks for your support! good job!
please fix based on the comments before we can merge

**Comment by gitstart** [comment]
> Thanks for your support! good job! please fix based on the comments before we can merge

Hi @yuval-hazaz We've resolved 1st and and 3rd feedback, for the 2nd one do we have to do attend to do that feedback ?

File: packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.scss (hunk 2 above)
Comment (yuval-hazaz) [reply packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.scss:144]: @EugeneTseitlin ?

**Review by yuval-hazaz — APPROVED** [review]

**Pull request merged**